Python/sign.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+
+# This script is a minimal example using a UpvestAPI object defined in
+# the ./upvest/http.py file in this project. The UpvestAPI class wraps
+# up the HTTP Message signature generation, based on the credentials
+# required as arguments to this script (a PEM file, a password for
+# that PEM file, the preshared Key ID (so we know what Public key to
+# validate your requests against!) and the Client ID and Client secret
+# required for the authorisation requests.
+
+# To use this script you'll need to:
+#
+#   pip3 install cryptography
+#   pip3 install requests
+
+
+import json
```

Comment:
Keep system imports together, and also above local imports.

```suggestion
import http.client as http_client
import json
import logging
```